### PR TITLE
Add Notebook terminal weblinks support

### PR DESCRIPTION
**Pull request by degrammer**
Prevent copy & paste for opening Notebook Terminal links. Allow click to open

support-weblinks

### CODE REVIEW

**Review by mxsdev — COMMENTED** [review]
See changes, other than that LGTM 👍

File: src/client/components/terminal.ts [hunk 1]
```diff
@@ -295,9 +296,9 @@ export class TerminalView extends LitElement {
     this.fitAddon = new FitAddon()
     this.fitAddon.activate(this.terminal!)
     this.serializer = new SerializeAddon()
-    const unicode11Addon = new Unicode11Addon()
     this.terminal.loadAddon(this.serializer)
-    this.terminal.loadAddon(unicode11Addon)
+    this.terminal.loadAddon(new Unicode11Addon())
+    this.terminal.loadAddon(new WebLinksAddon(this.#onWebLinkClick))
```
[the remark this answers is not in the document]
Comment (degrammer) [reply src/client/components/terminal.ts]: Thanks, fixed!

File: src/constants.ts [hunk 2]
```diff
@@ -23,7 +23,8 @@ export enum ClientMessages {
   terminalStdout = 'terminal:stdout',
   terminalStderr = 'terminal:stderr',
   terminalStdin = 'terminal:stdin',
-  activeThemeChanged = 'theme:changed'
+  activeThemeChanged = 'theme:changed',
+  openLink = 'terminal:openlink'
```
[the remark this answers is not in the document]
Comment (degrammer) [reply src/constants.ts]: Fixed!

**Review by mxsdev — APPROVED** [review]
LGTM

**Pull request merged**